cardazim/server.py

```diff
@@ -0,0 +1,67 @@
+import argparse
+import sys
+import struct
+import socket
+import threading
+from listener import Listener
+from card import Card
+import os
+
+lock = threading.Lock()
+card_num = 0
+
+def handle_connection(connection, path):
+    '''
+    Handle the connection with a client.
+    '''
+    global card_num
+    data = connection.receive_message()
+    card = Card.deserialize(data)
+    print("Received card.")
+    connection.close()
+    with lock:
+        n = card_num
+        card_num += 1
```

Comment:
I think that these names make the code more readable, what do you think?

```suggestion
        current_card_number = number_of_cards
        number_of_cards += 1
```